docs/blog/2018-09-24-reach-router/index.md

```diff
@@ -0,0 +1,65 @@
+
+Web apps rerender in the client -- without making a request to the server to fetch new HTML -- resulting in a faster, more seamless user experience. The performance benefits associated with not having to fully load a new page, however, can come at the expense of a broken experience for users who rely on assistive technology like screen readers.
+
+When navigating between traditional server-rendered pages, the page is fully reloaded and in response, screen readers can announce the new content. When using most client-side routing solutions (out of the box), without the page reload, screen readers don’t know new content has been loaded to focus or announce.
+
+The video below demonstrates this challenge (Video by [Rob DeLuca](https://twitter.com/robdel12), which accompanied his related article, “[Single page app routers are broken](https://medium.com/@robdel12/single-page-apps-routers-are-broken-255daa310cf)”)
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/NKTdNv8JpuM?rel=0" frameborder="0" allow="autoplay; encrypted-media" allowfullscreen></iframe>
+
+A primary focus (no pun intended) of @reach/router is to manage focus in client-side routing, out of the box, lifting the onus from devs to manage it from scratch. From the @reach/router documentation:
+
+> Whenever the content of a page changes in response to a user interaction, the focus should be moved to that content; otherwise, users on assistive devices have to search around the page to find what changed–yuck! Without the help of a router, managing focus on route transitions requires a lot effort and knowledge on your part.
+
+> Reach Router provides out-of-the-box focus management so your apps are significantly more accessible without you breaking a sweat.
+
+> When the location changes, the top-most part of your application that changed is identified and focus is moved to it. Assistive devices then announce to the user the group of elements they are now focused on, similarly to how it works when they load up a page for the first time.
+
+The video below demonstrates this focus management:
+
+<blockquote class="twitter-tweet" data-conversation="none" data-lang="en"><p lang="en" dir="ltr">Check out that focus management 😍<br><br>The same code that makes this possible is what makes relative links and embedded routers possible too. <a href="https://t.co/DjqveMfspA">pic.twitter.com/DjqveMfspA</a></p>&mdash; Ryan Florence (@ryanflorence) <a href="https://twitter.com/ryanflorence/status/1002219535921889281?ref_src=twsrc%5Etfw">May 31, 2018</a></blockquote>
+
+In terms of the development experience with Gatsby, this change is mostly under the hood, folded into the implementation of the Gatsby Link component. In terms of usability, accessibility by default is a win for everyone 🙌🏻.
+
```

Comment:
Any details on the other two improvements?  (Smaller bundle size, simplified API)

Reply:
Defer to @KyleAMathews on this. I could put the smaller bundle size in the context of the overall v1 -> v2 size decrease. on simplified API, I never really used react-router, I'm taking that from his PR/the announcement post. I'm not sure what going into details on the API differences would add here.

Reply:
@shannonbux @KyleAMathews Going to resolve this, unless further comment?

Reply:
ok with me!